topo/sparse/sparse.pyx

```diff
@@ -11,7 +11,7 @@ the sparse matrix are available.
 
 from cython.operator cimport dereference as deref
 from cpython cimport bool
-import numpy
+import numpy, scipy
 cimport numpy
```

Comment:
I haven't yet heard whether this new dependence on scipy is required for all sparse projections.  If it is, we now depend on scipy, which we've been trying to avoid.  If it isn't, then please move the scipy import into the toSparseArray function and document that this particular function requires SciPy.